### PR TITLE
updating starter kit

### PR DESCRIPTION
**Pull request by sciabarracom**
Cleanup of the actionloop starter kit

### CODE REVIEW

**Comment by mrutkows** [comment]
@csantanapr @sciabarracom  Please do not approve PRs with Travis breaks.  I have had to fix Travis 3x in the last 2 weeks because we have been approving PRs where scancode breaks.  It should be a simple matter to read the logs and fix a few files that are causing the break.

File: actionloop-starter-kit/tests/src/test/scala/runtime/actionContainers/SingleTest.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ 
+  it should "return an echo of the input" in {
+    val (out, err) = withActionContainer() { c =>
+      val (initCode, _) = c.init(initPayload(code))
+      initCode should be(200)
+      val (runCode, runRes) = c.run(runPayload(data))
+      runCode should be(200)
+    }
+  }
+}
```
Comment (mrutkows) [inline actionloop-starter-kit/tests/src/test/scala/runtime/actionContainers/SingleTest.scala]: Please fix Travis EOL scancode break.

**Comment by sciabarra** [comment]
I am sorry I did not notice the breakage of the build. Will fix asap

**Comment by csantanapr** [comment]
If Travis build is broken it should block PRs from being merge. If it’s not working like this let’s fix it.

**Comment by sciabarracom** [comment]
Now the build is clean, and the changes are minimal mostly to keep in sync with the tutorial.

**Comment by mrutkows** [comment]
@sciabarra Thanks Michele!

**Pull request merged**